open_academy/controllers/main.py

```diff
@@ -0,0 +1,7 @@
+from odoo import http
+
+class DashboardController(http.Controller):
+
+    @http.route('/web/view/edit_custom', type='json', auth="user")
+    def edit_custom(self, arch):
```

Comment:
Please add a comment, why are you doing this, just to have a reason.

Reply:
Changes were applied!